ParisBNF/abb/BNFabb164.xml

```diff
@@ -345,7 +345,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                            <material key="parchment"/>
                         </support>
                         <extent>
-                           <measure unit="leaf">i + 206</measure>
+                           <measure unit="leaf" quantity="207">i + 206</measure>
```

Comment:
The guidelines have "To record the presence of protective leaves, please use a syntax involving only Arabic numerals, like 3+69+1 to indicate that there are three guard leaves at the beginning and another one at the end."
I am not sure how important it is not to have spaces in the sequence?

Reply:
I do not remember now whether the script calculates the values in the text area at the moment, but in case we ever might need it it is surely better to follow the guidelines. For a computer, 1+207 means 208. i +  207 or even 1 + 207 does not mean anything.

Reply:
How to do if the first folia were foliated with Roman numbers? For instance if it is i-iii-200, should it be written 3+200, and 
a separate explanation that 3 is "i-ii-iii"?

Reply:
I think the foliation should be better explained under `foliation`, `measure `is about numbers